final_prep/annotations.rb

```diff
@@ -3,11 +3,15 @@
 # Use the # to create a new comment
```

Comment:
I gave you a lot of comments here BUT it's because I can see you have the general idea of what's happening and want to push you a bit. Tighten up the vocabulary you use, and if that's a challenge that's good - it's an opportunity to check yourself a bit on your understanding of these things. Overall, I think you're in a good spot based on this.